kanvas/src/androidMain/kotlin/ScaledIsPointInPath.kt

```diff
@@ -0,0 +1,45 @@
+package com.juul.krayon.kanvas
+
+import android.graphics.Matrix
+
+public class ScaledIsPointInPath(
+    private val scalingFactor: Float = 1f
+) : IsPointInPath {
+
+    override fun isPointInPath(transform: Transform, path: Path, x: Float, y: Float): Boolean {
+        val nativePath = path.toAndroid().apply {
+            val matrix = Matrix().applyTransform(transform)
+            transform(matrix)
+        }
+        val left = x / scalingFactor - 0.002f
+        val top = y / scalingFactor - 0.002f
+        val right = x / scalingFactor + 0.002f
+        val bottom = y / scalingFactor + 0.002f
```

Comment:
Am curious what is the reason for the fudge factor? Also, perhaps should go in a named `const` which, if named well, may have been enough to prevent me asking what it is for... :-)

Reply:
Will add a comment. tl;dr is Android doesn't let you query for is-point-in-path, but does let you query for do-these-shapes-overlap. So we have to give it a rectangle of non-zero area, because a zero-area rectangle will never "overlap".